src/openMVG/matching/regions_matcher.hpp

```diff
@@ -222,14 +223,45 @@ class RegionsMatcherT : public RegionsMatcher
       vec_fDistance.begin(), // distance start
       vec_fDistance.end(),   // distance end
       NNN__, // Number of neighbor in iterator sequence (minimum required 2)
-      vec_nn_ratio_idx, // output (indices that respect the distance Ratio)     
+      vec_valid_matches_idx, // output (indices that respect the distance Ratio)     
       b_squared_metric_ ? Square(f_dist_ratio) : f_dist_ratio);
 
-    vec_putative_matches.reserve(vec_nn_ratio_idx.size());
-    for (size_t k=0; k < vec_nn_ratio_idx.size(); ++k)
+    const size_t n = vec_fDistance.size();
+    
+    // Consider the match valid if the 2 first matches correspond to the "same" feature.
+    //
+    // In [Mishkin 2015] "MODS: Fast and Robust Method for Tow-View Matching" they introduce the FGINN strategy
```

Comment:
Tow -> tow
@fcastan @simogasp 
I think this modification is very dangerous! Since it can really change the behavior.
It could be activated thanks to a boolean or an option... And could be done with a decorator over the existing code.

I think this modification is really efficient for MODS since they are synthesizing some virtual views, and so feature duplication become common...

When features are very sparse, I think this test can drop out all the correspondences!

Moreover the threshold must be perhaps dependent of the image size?